src/utils/api.js

```diff
@@ -26,13 +26,22 @@ const handleResponse = (request) =>
     },
   )
 
+export const LOCALE = {
+  EN: 'en',
+  DE: 'de',
+  EL: 'el',
+  ES: 'es',
+  FR: 'fr',
+  HE: 'he',
+  IT: 'it',
+  NL: 'nl',
+  PL: 'pl',
+}
```

Comment:
Great use of an object to represent an enum! It's best to freeze this object. See here: https://stackoverflow.com/questions/287903/what-is-the-preferred-syntax-for-defining-enums-in-javascript